window/debug.c

```diff
@@ -48,7 +48,7 @@ void Window_LogWrite(const char *fmt, ...)
 
 #else
 
-void Window_LogFile(char *filename) { }
-void Window_LogWrite(char *fmt, ...) { }
+void Window_LogFile([[maybe_unused]]char *filename) { }
+void Window_LogWrite([[maybe_unused]]char *fmt, ...) { }
```

Comment:
What's the purpose of adding `[[maybe_unused]]` here?

Reply:
These stops the compiler from giving warning about unused variables. These variables are unused under normal operations, but enabled when enabling `DEBUG`.
See also https://en.cppreference.com/w/cpp/language/attributes/maybe_unused

Reply:
Ahh I see, I didn't realise the functions were still called if debugging was disabled.  Thanks for the explanation.